modules/nw-networkpolicy-configuring-ingress-new-deployment.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/network_policy/nw-networkpolicy-full-multitenant-isolation.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-networkpolicy-ingress-new-deployments_{context}"]
+= Creating a network policy for new projects
+
+After you have created a network policy that defines specific connections, new pods within a project are unable to communicate with existing pods in the same project, until the proper network policy has been applied to that pod. To address this behavior, you can create the following `allow-ingress-from-new` and `allow-ingress-to-new` network policies in a project, which allows new pods with the `networking/allow-all-connections=true` label to communicate with existing pods until more granular policies are created. 
+
+.Prerequisites
+
+* You have created the `deny-by-default` network policy and applied it to a project.
+
+.Procedure
+
+. Create a new project, for example, `project-c`, by running the following command:
++
+[source,terminal]
+----
+$ oc new-project project-c
+----
+
+. In the `project-c` namespace, create a new pod by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f - -n project-c
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-pod
+  labels:
+    app: busybox
+spec:
+  containers:
+  - name: busybox
+    image: alpine:latest 
+    command: [ "sleep", "3600" ]
+    securityContext:
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - "ALL"
+      seccompProfile:
+        type: "RuntimeDefault"
+EOF
+----
+
+. In the `project-a` namespace:
+
+.. Create the following `allow-ingress-from-new` network policy, which allows pods in this project the ability to receive ingress from a new project:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-ingress-from-new
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          networking/allow-all-connections: "true"
+----
```

Comment:
If the intent is to allow the ingress from new project-c then the policy needs to be as below in project-a
<pre>
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: allow-ingress-from-new
spec:
  ingress:
  - from:
    - namespaceSelector: {}
      podSelector:
        matchLabels:
          networking/allow-all-connections: "true"
  podSelector: {}
  policyTypes:
  - Ingress

</pre>

Reply:
Updated.